src/supercontrast/metrics/calculators/bleu_calculator.py

```diff
@@ -0,0 +1,39 @@
+import nltk
+
+from nltk.tokenize import word_tokenize
+from nltk.translate.bleu_score import sentence_bleu as nltk_sentence_bleu
+from sacrebleu import sentence_bleu
+
+from supercontrast.metrics.metrics_calculator import MetricsCalculator
+from supercontrast.metrics.metrics_enum import Metric
+
+
+class BLEUCalculator(MetricsCalculator[str, float]):
+    def __init__(self, *args, **kwargs):
+        super().__init__(Metric.BLEU, *args, **kwargs)
+
+    def calculate(self, reference: str, hypothesis: str) -> float:
+        return sentence_bleu(hypothesis, [reference]).score
+
+    def get_name(self) -> str:
+        return "BLEU Score"
+
+
+class BLEUNLTKCalculator(MetricsCalculator[str, float]):
```

Comment:
why is this separate from the BLEU calculator?

Reply:
I believe the NTLK score uses a tokenization method that's more consistent, we can keep one as ground truth if you think that's better but I figured there was no harm in both